alf/networks/preprocessor_networks.py

```diff
@@ -68,12 +68,12 @@ def __init__(self,
         self._input_preprocessor_modules = nn.ModuleList()
 
         def _get_preprocessed_spec(preproc, spec):
```

Comment:
spec is no longer needed

Reply:
I didn't quite understand your comment. This is getting the preprocessed spec from the original spec. You mean this is step is not needed?

Reply:
Ignore the comment above. What if preproc is a layer as in Wei's carla gin file? The if condition here basically excludes the possibility of using non-network modules as preprocessors?

Reply:
Generic layer or nn.Module is not supported currently.
I guess something like a SequentialNetwork which can wrap layers/modules into a Network could be used in the case you mentioned.

Reply:
I think then it's best to directly assert False in the case of non-Network instead of doing something that the user isn't aware of, or potentially crash the code when the return spec is incorrect.

Reply:
Good point. Now added.